Suggest user to use WSL in Windows instructions (#2867)
cocotb/cocotb: documentation/source/install.rst

   .. group-tab:: Windows

>       We recommend users who are running Windows, and who are more comfortable with a Unix shell or who have legacy Makefile-based projects,
>       to use either Windows Subsystem for Linux (WSL) or a Virtual Machine, such as Oracle VirtualBox.

Author: ```suggestion
      We recommend users who are running Windows and who are more comfortable with a Unix shell or who have legacy Makefile-based projects to use either Windows Subsystem for Linux (WSL) or a Virtual Machine such as Oracle VirtualBox.
```
Better grammar I think? @cmarqu

Reviewer: Uh, not sure really... based on feeling I might even add a comma before the "or who have".

>       We recommend users who are comfortable with a Unix shell and who are running Windows 10 or 11 use Windows Subsystem for Linux (WSL).
>       See `here <https://docs.microsoft.com/en-us/windows/wsl/install>`_ for instructions on installing WSL,
>       then follow the Linux installation instructions for cocotb.

Author: @imphil Is this any better?

```suggestion
      We recommend users who are running Windows, and who are more comfortable with a Unix shell or who have legacy Makefile-based projects
      to use either Windows Subsystem for Linux (WSL) or a Virtual Machine, such as Oracle VirtualBox.
      After installing WSL or a Virtual Machine and a supported Linux distribution, follow the Linux installation instructions for cocotb.
```

>       We recommend users who are comfortable with a Unix shell and who are running Windows 10 or 11 use Windows Subsytem for Linux (WSL).
>       See `here <https://docs.microsoft.com/en-us/windows/wsl/install>`_ for installation instructions of WSL,
>       then follow the Linux installation instructions for cocotb.

Reviewer 2: ```suggestion
      We recommend users who are comfortable with a Unix shell and who are running Windows 10 or 11 use Windows Subsystem for Linux (WSL).
      See `here <https://docs.microsoft.com/en-us/windows/wsl/install>`_ instructions on installing WSL,
      then follow the Linux installation instructions for cocotb.
```

Reviewer 2: hah, yeah

>       `Conda <https://conda.io/>`_ is an open-source package and environment management system that we recommend for users who are more comfortable with Windows.

Author: ```suggestion
      `Conda <https://conda.io/>`_ is an open-source package and environment management system that we recommend for users who are more comfortable with native Windows development.
```

   .. group-tab:: Windows

      We recommend users who are running Windows and who are more comfortable with a Unix shell,
      or who have legacy Makefile-based projects,
      to use Windows Subsystem for Linux (WSL).

>       After installing WSL or a Virtual Machine and a supported Linux distribution, follow the Linux installation instructions for cocotb.

Author: ```suggestion
      After installing WSL and a supported Linux distribution, follow the Linux installation instructions for cocotb.
```
